lib/data/coastline_coverage/coastline_coverage.csv

```diff
@@ -1,613 +1,640 @@
+"SJM",8542.94482654175,0,0,0,0,131.867542578041,0,50
+"SLB",9758.29128195567,0,0,0,309.50647329846,0,9429.34223789496,50
+"SLE",2276.75527691923,332.586474955633,0,0,0,0,1940.46731461391,50
+"SLV",767.263124070043,766.60045447032,0,0,0,0,0,50
+"SOM",3720.66624320059,247.327478581553,0,117.066297900972,434.657359334671,260.941667874308,948.025868990306,50
+"SPM",182.804422913431,0,0,0,0,0,0,50
+"STP",265.998253133533,0,0,0,0,0,174.246919677303,50
+"SUR",1250.89677270097,728.762694902362,0,0,0,0,487.798260499465,50
+"SVN",46.6647411501973,0,0,0,0,0,46.7402955232278,50
+"SWE",23099.4861910245,0,8777.40788274165,418.729783703845,0,0,13548.8940615066,50
+"SXM",29.3101103763941,0,0,0,0,0,29.2603452379494,50
+"SYC",762.619501940772,0,0,0,129.563772978354,0,627.29533758493,50
+"SYR",213.7015488661,0,0,213.529242094489,0,0,0,50
+"TCA",801.56204862849,0,0,0,0,0,800.40717533227,50
+"TGO",57.5681175179602,0,0,14.6834383981687,0,0,42.8913759780555,50
+"THA",8815.71300755695,562.029698353061,0,0,110.734858899698,0,8120.07894485815,50
+"TKL",137.828877470597,0,0,0,137.516455543682,0,0,50
+"TKM",1643.45707194602,0,0,1098.26489393906,0,0,0,50
+"TLS",784.271786814486,29.9661600146639,0,0,32.7598456617945,0,720.795531573583,50
+"TON",855.42294663978,0,0,0,103.125961016137,0,724.6963038791,50
+"TTO",692.980765096849,22.0382980943406,0,0,0,0,669.896773097853,50
+"TUN",1950.65118029227,0,0,968.820891025394,0,0,981.831114977413,50
+"TUR",7857.96592217332,0,1274.57814858407,1224.56576911189,0,0.579382558061456,4013.87142779661,50
+"TUV",245.876251246892,0,0,0,245.199984729829,0,0,50
+"TWN",2380.04297044105,35.1913678546738,178.830951860678,0,737.794964719944,0,1403.17661944919,50
+"TZA",3911.13523718508,57.2636909407341,0,0,0,0,3712.30291072953,50
+"UKR",4969.13982696616,0,431.750627494061,3005.32978981898,0,0,952.752788826616,50
+"UMI",208.569656743173,0,0,0,143.053787197887,0,65.2593282675697,50
+"URY",1660.47004549625,0,423.324610050668,0,0,34.0621054822777,157.426170112757,50
+"USA",158095.958705677,6.89497331485344,47415.6343281327,3201.73864732045,0,6188.81799259785,91879.5698147613,50
+"VCT",306.304661723734,0,0,0,0,0,305.679309611848,50
+"VEN",11992.676060775,6226.03812226159,0,0,0,5.13059046651837,4300.78544973144,50
+"VGB",328.875116838527,0,0,0,0,0,328.503410614391,50
+"VIR",364.342620623377,0,0,0,0,0,363.919780968217,50
+"VNM",15006.3356871535,2029.68656913059,582.595436980978,10.5560463527137,756.061874753668,0,10712.211795523,50
+"VUT",3049.69943579994,0,0,0,871.461883557195,0,2172.37060455036,50
+"WLF",302.31477979448,0,0,0,0,0,301.733369469651,50
+"WSM",449.302732232574,0,0,0,0,0,448.766639055566,50
+"xHT",309.936958415584,0,0,0,71.9469023663863,0,237.560725624442,50
+"xKI",1263.82808857718,0,0,285.107675391024,0,120.165150203047,104.191896209455,50
+"xPI",155.148819782731,0,0,0,133.450474485418,0,21.6099649032627,50
+"xSI",18.8032133103445,0,0,0,17.2318494174414,0,1.54370852231876,50
+"xSK",19.0163705162395,0,0,0,19.0107420287208,0,0,50

```

Comment:
I'm guessing these are just ignored, seeing as they are not real ISO3 codes (as far as I'm aware)? And I guess the second question is, should they be ignored?

Reply:
I can check this with CLS - this could potentially be some kind of non-ISO3 code (but still ISO) - best to keep this in case we need it?

Reply:
That would be great if you don't mind. I'm happy to merge this anyway as I don't think it's doing any harm, may just mean they need to supply a new csv if this data is meant to match up with a GeoEntity that doesn't currently exist/has a different code.